FIX Use composer update for prefer-lowest

emteknetnz opened the pull request:
Fixes this https://app.travis-ci.com/github/silverstripe/silverstripe-framework/jobs/567064712

Happened after we optimised to use `composer install` instead of `composer update` https://github.com/silverstripe/silverstripe-travis-shared/pull/35

dhensby commented:
Well, we can see that the `install` vs `update` hasn't really made any speed difference, which was the main aim of #35. It may just be more reliable to simply use `update` everywhere like we were before?

emteknetnz committed:
FIX Use composer update for prefer-lowest

emteknetnz commented:
Yup [install](https://app.travis-ci.com/github/silverstripe/silverstripe-framework/jobs/567571400#L466) is functionally the same speed as [update](https://app.travis-ci.com/github/silverstripe/silverstripe-framework/jobs/566923279#L466) - I've updated the PR to revert back to `composer update` and added some comments for explanation
